src/stdlib.js

```diff
@@ -196,6 +202,86 @@ defunTyped('not', ['boolean'], ['boolean'], function (stack, scope) {
 
 // Non-standard functions
 
+/* This function clones an object. Not sure if it belongs here, I will probably change this later.
+See: http://stackoverflow.com/questions/728360/most-elegant-way-to-clone-a-javascript-object */
+
+function clone(obj) {
```

Comment:
This is overkill. We don't need to clone arrays anywhere, and functions exist for that. We also don't need to do a deep copy, since objects would be immutable. Are you not very proficient with JavaScript?